app/jobs/reservation_job.rb

```diff
@@ -0,0 +1,44 @@
+class ReservationJob < ActiveJob::Base
+  queue_as :default
+
+  def perform
+    if enabled
+      log_start
+      run
+      clean
+      log_completion
+    else
+      log_disabled
+    end
+  end
+
+  private
+
+  def enabled
+    true
+  end
+
+  def log_start(type: '', task: '')
+    Rails.logger.info "Found #{collection.count} #{type} reservations, #{task}"
+  end
+
+  def log_disabled(task: '')
+    Rails.logger.info "Reservations is not configured to #{task}. "\
+      'Please change the application settings if you wish to do so.'
+  end
+
+  def log_completion
+    Rails.logger.info 'Done!'
+  end
+
+  def run
```

Comment:
I definitely think that either `run` needs to be blank by default or `enabled` needs to be set to `false`. I think that logging improperly set up tasks could be beneficial, though.